cli/src/commands/working-groups/verifyValidator.ts

```diff
@@ -0,0 +1,39 @@
+import { flags } from '@oclif/command'
+import WorkingGroupsCommandBase from '../../base/WorkingGroupsCommandBase'
+import { WorkingGroups } from '../../Types'
+
+export default class MembershipValidatorAccountCommand extends WorkingGroupsCommandBase {
+  static description = 'Membership lead/worker verifies validator membership profile'
+  static flags = {
+    message: flags.string({
+      required: true,
+      description: 'Memberhip ID of the validator to verify',
+    }),
+
+    ...WorkingGroupsCommandBase.flags,
+  }
+
+  async run(): Promise<void> {
+    const { message } = this.parse(MembershipValidatorAccountCommand).flags
+
+    const members = await this.getApi().groupMembers(WorkingGroups.Membership)
+    const lead = await this.getApi().groupLead(WorkingGroups.Membership)
+    // const lead = await this.getRequiredLeadContext()
+    const pairs = this.getPairs()
+
+    const membersRows = members.filter((m) =>
+      pairs.find((p) => p.address === m.roleAccount.toString()) ? m.memberId.toString() : undefined
+    )
```

Comment:
You can use [getRequiredWorkerContext](https://github.com/chrlschwb/joystream/blob/17b39cf69fc297fab10ab6528b4ee5b67441b581/cli/src/base/WorkingGroupCommandBase.ts#L50) helper function here to get the given worker